srcpkgs/fscrypt/template

```diff
@@ -1,14 +1,27 @@
 # Template file for 'fscrypt'
 pkgname=fscrypt
 version=0.2.6
-revision=1
-build_style=go
-go_import_path=github.com/google/fscrypt
-go_package="${go_import_path}/cmd/fscrypt"
-makedepends="pam-devel"
+revision=2
+build_style=gnu-makefile
+makedepends="go m4 pam-devel"
```

Comment:
only `pam-devel` is `makedepends`, `m4` and `go` belong to `hostmakedepends`.

Reply:
Just took a better look at `Manual.md`, and fixed it.